pillar/heroku/bootcamps.sls

```diff
@@ -138,6 +141,9 @@ heroku:
     NOVOED_API_BASE_URL: https://api.novoed.com/
     NOVOED_API_KEY: __vault__::secret-operations/global/{{ business_unit }}/novoed>data>api_key
     NOVOED_API_SECRET: __vault__::secret-operations/global/{{ business_unit }}/novoed>data>api_secret
+    NOVOED_SAML_DEBUG: {{ env_data.NOVOED_SAML_DEBUG }}
+    NOVOED_SAML_KEY: __vault__::secret-operations/global/{{ business_unit }}/novoed>data>saml_key
+    NOVOED_SAML_CERT: __vault__::secret-operations/global/{{ business_unit }}/novoed>data>saml_cert
```

Comment:
Since the key and cert are different for RC/CI and Prod, I'd change the paths to this:
`__vault__::secret-{{ business_unit }}/{{ env_data.vault_env_path }}/novoed>data>saml_{key,cert}`

Reply:
@gsidebo `global` shouldn't be in the path since it's env dependent. I would just change it to look exactly like the path I listed

Reply:
Whoops! Good catch. Fixed now